docs/guide/01-api-protobuf.md

```diff
@@ -111,46 +132,61 @@ Java:
   public String getSongName() { ... }
   public Builder setSongName(String v) { ... }
 ```
-#### 数组 Repeated
-通过repeated关键字定义数组（List）：
+
+如果你的字段名包含一个数字，数字应该出现在字母后面，而不是下划线后面。例如，使用 `song_name1` 而不是 `song_name_1`。
+
+### Repeated 字段
+
+对于复数命名的 message 使用 repeated 字段。
+
 ```protobuf
 repeated string keys = 1;
 ...
 repeated Account accounts = 17;
 ```
 
-#### 枚举 Enums
-使用驼峰命名法（首字母大写）命名枚举类型，使用 “大写_下划线_大写” 的方式命名枚举值：
+### 枚举 Enums
+
+枚举类型使用驼峰命名法（首字母大写）命名，使用 `CAPITALS_WITH_UNDERSCORES(带下划线的大写字母)` 的方式命名枚举值：
+
 ```protobuf
-enum Foo {
-  FIRST_VALUE = 0;
-  SECOND_VALUE = 1;
+enum FooBar {
+  FOO_BAR_UNSPECIFIED = 0;
+  FOO_BAR_FIRST_VALUE = 1;
+  FOO_BAR_SECOND_VALUE = 2;
 }
 ```
-每一个枚举值以分号结尾，而非逗号。
 
-#### 服务 Services
-如果你在 .proto 文件中定义 RPC 服务，你应该使用驼峰命名法（首字母大写）命名 RPC 服务以及其中的 RPC 方法：
+每一个枚举值以分号结尾，而非逗号。倾向于给枚举值加前缀，而不是用一个包围的信息包围它们。零值枚举应该有后缀 `UNSPECIFIED`。
+
+### 服务 Services
+
+如果你在 `.proto` 文件中定义 RPC 服务，你应该使用驼峰命名法（首字母大写）命名 RPC 服务以及其中的 RPC 方法：
+
 ```protobuf
 service FooService {
-  rpc GetSomething(FooRequest) returns (FooResponse);
+  rpc GetSomething(GetSomethingRequest) returns (GetSomethingResponse);
+  rpc ListSomething(ListSomethingRequest) returns (ListSomethingResponse);
 }
 ```
 
-### Comment
-* Service，描述清楚服务的作用
-* Method，描述清楚接口的功能特性
-* Field，描述清楚字段准确的信息
+## Comment
+
+- **Service**，描述清楚服务的作用
+- **Method**，描述清楚接口的功能特性
+- **Field**，描述清楚字段准确的信息
+
+## Examples
+
+API Service 接口定义(demo.proto)
 
-### Examples
-API Service接口定义(demo.proto)
 ```protobuf
 syntax = "proto3";
 
 package kratos.demo.v1;
 
 // 多语言特定包名，用于源代码引用
-option go_package = "github.com/go-kratos/kratos/demo/v1;v1";
```

Comment:
the name after ; is for relative code generation, should not be removed

Reply:
OK. I understands. Shoud I need add this comment to the code?

Reply:
I'm update it and add the comment. If you think the comment need to be removed. pls comment it. Thx!